tests/hikari/impl/test_stateless_event_manager.py

```diff
@@ -95,7 +95,7 @@ async def test_on_channel_update(self, event_manager, shard, payload):
         await event_manager.on_channel_update(shard, payload)
 
         event_manager.dispatch.assert_awaited_once_with(event)
-        event_manager._app.event_factory.deserialize_channel_update_event.assert_called_once_with(shard, payload)
+        event_manager._app.event_factory.deserialize_channel_update_event.assert_called_once_with(shard, payload, None)
```

Comment:
These tests should be changed to also make sure they fetch from the cache. This will require mocking `._cache` and then making sure they are getting called with what you expect

Reply:
Nvm, just realized this is the stateless one. :sweat_smile:.

But we would still need the event_factory tests to make sure the field gets set